packages/graph-explorer/src/modules/NodesStyling/NodeStyleDialog.tsx

```diff
@@ -179,11 +179,12 @@ function Content({ vertexType }: { vertexType: string }) {
             asChild
           >
             <IconButton
-              variant="filled"
-              className="text-text-primary hover:text-text-primary group rounded-full border-0 bg-transparent p-0 hover:cursor-pointer hover:bg-gray-200"
+              variant="default"
+              className="group rounded-full"
+              style={vertexStyle}
```

Comment:
```suggestion
              variant="default"
              className="group rounded-full"
```

Great change! For the upload icon, let's let the default button color shine through (black). So I suggest removing the `style={vertexStyle}`